x-pack/plugins/triggers_actions_ui/public/application/components/builtin_action_types/email/email.tsx

```diff
@@ -75,6 +75,26 @@ export function getActionType(): ActionTypeModel<EmailActionConnector, EmailActi
           )
         );
       }
+      if (action.config.hasAuth && !action.secrets.user && !action.secrets.password) {
+        errors.user.push(
+          i18n.translate(
+            'xpack.triggersActionsUI.components.builtinActionTypes.error.requiredAuthUserNameText',
+            {
+              defaultMessage: 'Username is required.',
+            }
+          )
+        );
+      }
+      if (action.config.hasAuth && !action.secrets.user && !action.secrets.password) {
```

Comment:
Should this check password only?

```suggestion
      if (action.config.hasAuth && !action.secrets.password) {
```

Reply:
I've added this checks to avoid a stack of errors under the one input like:
'Username is required.'
'Username is required when password is used.'
But if it is ok, then I can remove it.

Reply:
Ohh I see what you mean, I just tested it. Thanks for the insight!